WordPressVIPMinimum/Sniffs/Files/IncludingFileSniff.php

```diff
@@ -104,6 +115,16 @@ public function process_token( $stackPtr ) {
 			return;
 		}
 
+		if ( $this->tokens[ $nextToken ]['code'] === T_CONSTANT_ENCAPSED_STRING && filter_var( str_replace( [ '"', "'" ], '', $this->tokens[ $nextToken ]['content'] ), FILTER_VALIDATE_URL ) ) {
+			$message = 'Include path must be local file source, external URLs are prohibited on WordPress VIP.';
+			$this->phpcsFile->addError( $message, $nextToken, 'ExternalURL' );
+			return;
+		}
+
+		if ( $this->has_custom_path( $nextToken ) !== false && array_key_exists( $this->tokens[ $nextToken ]['content'], $this->restrictedConstants ) === false ) {
```

Comment:
* Why are you checking against `!== false` when the method will only ever return `true` or `false` ?
* Why use `array_key_exists()` instead of `isset()` ?
* Why are these conditions in this order ?